galasa-extensions-parent/dev.galasa.ras.couchdb/build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 
 description = 'Galasa RAS - CouchDB'
 
-version = '0.25.0'
+version = '0.26.0'
```

Comment:
shouldn't this be 28?

Reply:
Addressed in https://github.com/galasa-dev/extensions/pull/200/commits/0124979801009e10b7c4fc6f1bce4ef8583e247b